### PR TITLE
configure: make ChakraCore default engine

### PR DESCRIPTION
**Pull request by obastemur**
##### Checklist

- [x] make ChakraCore default engine
- [x] update README file build instructions + 80 colon.
- [x] add additional information on Fedora prerequisites

### CODE REVIEW

**Comment by obastemur** [comment]
Fixes #167

File: node.gyp [hunk 1]
```diff
@@ -614,7 +614,7 @@
                 '-Wl,--no-whole-archive',
               ],
             }],
-          ]		       
+          ]
```
Comment (kunalspathak) [inline node.gyp:617]: It might not matter, but why not update on line 22 above and in `common.gypi` as well for completeness?

Comment (obastemur) [reply node.gyp:617]: In order to keep the difference minimum. We set it through configure anyways

**Review by kunalspathak — APPROVED** [review]

**Comment by obastemur** [comment]
@kunalspathak Thanks for the review

**Pull request merged**